packages/samples/headless-react/package.json

```diff
@@ -28,7 +28,6 @@
     "web-vitals": "1.1.2"
   },
   "scripts": {
-    "prepublishOnly": "exit 1",
```

Comment:
Why remove this ? It was there so that no one/no script accidentally publishes it.

Reply:
It's private now so prepublish won't be run